### DISCUSSION TITLE
Invalid package version found for Jackson Jakarta RS: JSON

### DISCUSSION THREAD

**Discussion by antoinetielbeke** [renovatebot/renovate#18868] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Our renovate finds a version for https://github.com/FasterXML/jackson-jakarta-rs-providers that does not exist.

```
DEBUG: Fetching changelog: https://github.com/FasterXML/jackson-databind (2.13.4 -> 2.13.4.2)
```
and later in the logs:
```json
            "sourceUrl": "https://github.com/FasterXML/jackson-databind",
            "versioning": "maven",
            "warnings": [],
            "updates": [
              {
                "bucket": "non-major",
                "newVersion": "2.13.4.2",
                "newValue": "2.13.4.2",
                "releaseTimestamp": "2022-10-13T17:33:39.000Z",
                "newMajor": 2,
                "newMinor": 13,
                "updateType": "patch",
                "pendingVersions": [
                  "2.14.0"
                ],
```

But on the [release page](https://github.com/FasterXML/jackson-jakarta-rs-providers/tags), there is not any `2.13.4.2`. 

Am I doing something wrong?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
you're mixing two different packages, your logs showing `jackson-databind`, which has that git version
